pages/admin/users.tsx

```diff
@@ -1,5 +1,95 @@
 import DashboardLayout from "components/DashboardLayout";
-import { useState } from "react";
+import { useEffect, useState } from "react";
+
+const UI_PAGE_SIZE = 7;
+const BACKEND_PAGE_SIZE = 21;
+
+const getBackendPageNumber = (uiPage: number): number[] => {
+  const pageNum = Math.floor((uiPage * UI_PAGE_SIZE) / BACKEND_PAGE_SIZE);
+  const startIndex = UI_PAGE_SIZE * (uiPage % 3);
+  return [pageNum, startIndex];
+};
+
+interface User {
+  name: string;
+  email: string;
+  image: string;
+  phoneNumber: string;
+}
+
+const UserDashboardItem: React.FunctionComponent<User> = ({
+  name,
+  email,
+  image,
+  phoneNumber,
+}) => {
+  return (
+    <div>
+      <div className="flex flex-row justify-between text-sm h-16 items-center">
+        <div className="flex flex-row justify-between">
+          <div className="w-10 h-10 mr-4 bg-placeholder rounded-full">
+            <img src={image} alt="" />{" "}
+            {/* Not being used right now because seed data doesn't have images */}
+          </div>
+          <div className="w-32">
+            <p className="font-display">{name}</p>
+            <p>User Role</p>
+          </div>
+        </div>
+        <div className="w-56">
+          <p>{email}</p>
+        </div>
+        <p>{phoneNumber}</p>
+      </div>
+      <hr className="border-unselected border-opacity-50" />
+    </div>
+  );
+};
+
+const UserDashboard: React.FunctionComponent = () => {
+  const [users, setUsers] = useState<User[]>();
+  const [index, setIndex] = useState(0);
+
+  const getUsers = async (pageNumber: number): Promise<void> => {
+    try {
+      const response = await fetch("http://localhost:3000/api/admin/users", {
+        method: "GET",
+        query: pageNumber,
```

Comment:
This line throws the error "Object literal may only specify known properties, and 'query' does not exist in type 'RequestInit' " when running yarn build